docker/spark-on-k8s/driver/Dockerfile

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Not really sure about the license.

Reply:
the license is inherit from the original spark-on-k8s dockerfiles.